### ISSUE TITLE
Revolutionary Rounds should immediately end if all head revs or all heads are dead & Exiled Head/Revs

### ISSUE THREAD

**Issue by keronshb** [space-wizards/space-station-14#21619]
## Description

I believe it _somewhat_ does this now but it should consider:

- The current amount of Heads & Rev Heads currently alive _on station_
- Any additions after the fact (if a CE joins mid round)
- If a Station Head or Rev Head are off station, they count as exiled which considers them dead, prevents stalling.

**Comment by 0x6273** [comment]
Why should round end immediately? Isn't it better to have shuttle be called, to give people time?

**Comment by keronshb** [comment]
> Why should round end immediately? Isn't it better to have shuttle be called, to give people time?

High Chaos round.

**Comment by lzk228** [comment]
> High Chaos round

lrp moment

**Comment by keronshb** [comment]
> > High Chaos round
> 
> lrp moment

Counts for NRP-MRP

**Comment by Carolyn3114** [comment]
> Why should round end immediately? Isn't it better to have shuttle be called, to give people time?

it's unfun to have the round go on when every security and head is dead, and the station's in chaos, it's also just awful for rp

**Comment by 0x6273** [comment]
> > Why should round end immediately? Isn't it better to have shuttle be called, to give people time?
> 
> it's unfun to have the round go on when every security and head is dead, and the station's in chaos, it's also just awful for rp

Yeah that's true. Maybe it should end instantly only if the revs win. And if all the head revs die, shuttle is called. Often the revs fluke, and it's disruptive to have the round just suddenly end in that case.
